src/chi/components/spinner/spinner.scss

```diff
@@ -0,0 +1,115 @@
+@import 'variables';
+@import 'mixins';
+
+$offset: 187;
+$duration: 1.4s;
+
+.chi {
+  .a-spinner__backdrop {
+    background: rgba(set-color(white), 0.7);
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    .a-spinner__icon {
+      animation: rotator $duration linear infinite;
+      width: 4rem;
+      height: 4rem;
```

Comment:
@davidojedacl set stroke: currentColor; as a property and makes tests with the different text color properties (-textBrand, -textSuccess...)